crates/sui-json-rpc/src/api.rs

```diff
@@ -496,10 +496,22 @@ pub trait EventReadApi {
     ) -> RpcResult<Vec<SuiEventEnvelope>>;
 }
 
-#[open_rpc(namespace = "sui", tag = "Quorum Driver APIs to execute transactions.")]
+#[open_rpc(namespace = "sui", tag = "APIs to execute transactions.")]
 #[rpc(server, client, namespace = "sui")]
-pub trait QuorumDriverApi {
-    /// Execute the transaction and wait for results if desired
+pub trait TransactionExecutionApi {
+    /// Execute the transaction and wait for results if desired.
+    /// Request types:
+    /// 1. ImmediateReturn: immediate return to client without waiting for any results
```

Comment:
immediately returns - is this the same as "Sends a response" or responds to the client? It is not clear what it "returns"

Reply:
yes, in this context "return" means sending the response to client. How would you recommend to make the text clear?

Reply:
Immediately returns a response to the client